series.tex

```diff
@@ -0,0 +1,79 @@
+% Series expansion (Differentiate between the two approaches being used)
+\subsubsection{Series Expansion}
+
+SymPy is able to calculate the symbolic series expansion of an arbitrary series
+or expression involving elementary and special functions and multiple
+variables. For this it has two different implementations.
+
+The first approach stores a series as a core object. Each function has its
+specific implementation of its expansion which is able to evaluate the Puiseux
+series expansion about any point.
+
+\begin{verbatim}
+>>> from sympy import symbols, series
+>>> x, y = symbols('x, y')
+>>> series(sin(x+y) + cos(x*y), x, 0, 2)
+1 + sin(y) + x*cos(y) + O(x**2)
+\end{verbatim}
+
+The newer and much faster approach called Ring Series makes use of the
+observation that a truncated Taylor series, is in fact a polynomial.
+Ring Series uses the efficient representation and operations of sparse
+polynomials. The choice of sparse polynomials is deliberate as it performs
+well in a wider range of cases than a dense representation. Ring Series gives 
+the user the freedom to choose the type of coefficients he wants to have in
+his series, allowing the use of faster operations on certain types.
+
+For this, several low level methods for expansion of elementary functions are
+implemented using semi-numerical algorithms. All these support Puiseux series
+expansion.
+
+\begin{verbatim}
+>>> from sympy import ring
+>>> from sympy.polys.ring_series import rs_sin
+>>> R, x = ring('x', QQ)
+>>> rs_sin(x**2 + x, x, 5)
+-1/2*x**4 - 1/6*x**3 + x**2 + x
+\end{verbatim}
+
+The function \texttt{rs\_series} makes use of these elementary functions to
+intelligently expand an arbitrary SymPy expression. Currently it only supports
+expansion about 0 and is under active development. Ring Series is several times
+faster than the default implementation with the speed difference increasing
```

Comment:
You could add a comment here. When you have the benchmarks you can add them here. @shivamvats